receiver/redisreceiver/metadata.yaml

```diff
@@ -13,6 +13,14 @@ resource_attributes:
     description: Redis server's version.
     enabled: true
     type: string
+  server.address:
+    description: Redis server's address
+    enabled: true
+    type: string
+  server.port:
+    description: Redis server's port
+    enabled: true
+    type: string
```

Comment:
Is the port a string?

Reply:
We could parse it to get the corresponding number, but it is a string when [it is taken from the address](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/26707/files#diff-6d24e7ef9f7296d9beed1c934158a4b6b1ef055abe3ea78ea78a0460a6e7f7f8R48)

Reply:
it's fine, let's leave it as is